### PR TITLE
Tkt 00088 - More HLM fixes

### PR DESCRIPTION
**Pull request by ishanivardhan**
- Part 1 > Healthcare Leadership Model – The Model > The importance of context
**Text aligned with checkboxes**

- Part 1 > Healthcare Leadership Model – The Model > The importance of context
**The screen would something like below, will also ask Tilly to add the words "Correct/Incorrect" next to Response**
screen shot 2018-07-04 at 14 09 53

- Part 2 > Facilitation skills > Facilitation and coaching skills
**Same as above**

-  Part 1 > Role of a Feedback Facilitator > 360 process – checking your understanding
(How to create Drag & Drop was the query)
**Will suggest to use Matching Question for this**

- Part 2 > Assessment
**Text aligned with Radio buttons**

- Part 2 > Assessment had some of the options bunched together so the alignment is slightly off.
**Edited the main text itself to remove extra <br> tags, that fixed it. No CSS required**

- Part 2 > Assessment, 6th option here which isn’t showing, it should be “All of the above”.
**Fixed**

- Part 2 > Assessment, Would it be possible to realign the “x” button here
**Modified Heading size which fixed the "x" button to fit in the popup**

- Part 2 > Assessment: The end page where it shows each question and what the user answered – and what the correct answer they should have chosen is. I’m not sure we want to be showing the correct answers as they will be able to go forward and do their 2nd attempt knowing what they were supposed to click.
**Fixed by changing settings under Quiz > Edit Settings > Review > Unchecked "Whether correct" & "Specific Feedback" boxes for **_Immediately after the attempt_** & **_Later, while the quiz is still open_** columns**

- When you click “finish review” at the end of the screen, would it be possible to show their highest grade in a % format rather than a decimal number?
**Tried changing few settings but didnt help so this one may have to stay as is.**

cc: @cehwitham Pls review when you can. Thanks!

### CODE REVIEW

File: style/middleware/_middleware.filepicker.scss [hunk 1]
```diff
@@ -88,6 +88,10 @@ $grid-breakpoints: (
   @extend .modal-title;
 }
 
+.moodle-dialogue-hd h1 {
+  font-size: $global-font-size-h2;
```
Comment (cehwitham) [inline style/middleware/_middleware.filepicker.scss:92]: Is there a reason we're making a h1 the size of a h2?

File: style/middleware/_middleware.quizactivity.scss [hunk 2]
```diff
@@ -41,4 +41,45 @@
...
+.answeroption .row .checkbox label input[type='checkbox'] {
+  float: left;
+  margin-top: $global-spacing-unit-tiny;
+  margin-left: $global-spacing-unit-tiny;
+}
+
+.answer label {
+  display: inline-block;
+  margin-left: $global-spacing-unit-tiny;
+  font-weight: normal;
```
Comment (cehwitham) [inline style/middleware/_middleware.quizactivity.scss:57]: Is this overriding a previous setting? Can we prevent this somehow?

File: style/middleware/_middleware.quizactivity.scss [hunk 3]
```diff
@@ -41,4 +41,45 @@
...
+}
+
+.generaltable tbody>tr:nth-child(odd)>td {
+  background-color: color('nhs-grey-pale');
+}
+
+.answer label {
+  display: inline-block;
+  font-weight: normal;
+  margin-left: 8px;
```
Comment (cehwitham) [inline style/middleware/_middleware.quizactivity.scss]: None standard spacing unit?

**Review by cehwitham — COMMENTED** [review]
Just a few questions.

File: style/middleware/_middleware.filepicker.scss (hunk 1 above)
Comment (ishanivardhan) [reply style/middleware/_middleware.filepicker.scss:92]: This is to fit the "x" close button within the popup. Currently, the Confirmation dialog/popup has inline style/width added to it from the code with an ID selector which I can't override, hence tweaking this the other way round by making the heading slightly smaller to make the pop-up look neat.

File: style/middleware/_middleware.quizactivity.scss (hunk 2 above)
Comment (ishanivardhan) [reply style/middleware/_middleware.quizactivity.scss:57]: There was no setting to this selector initially and the text was showing up as bold and not aligned with checkboxes. Especially the text that spanned over two lines due to 2-col layout widths. In order to align them, I had to add some of these styles. Making the text non-bold just makes it look neater but can remove that bit if you like. 

I have tested the responsiveness for this and seems to be fine.

File: style/middleware/_middleware.quizactivity.scss (hunk 3 above)
Comment (ishanivardhan) [reply style/middleware/_middleware.quizactivity.scss]: That's replaced with SASS variable, commit pushed.

**Review by cehwitham — APPROVED** [review]

**Pull request merged**